appinventor/components/src/com/google/appinventor/components/runtime/Web.java

```diff
@@ -929,6 +1006,7 @@ private static HttpURLConnection openConnection(CapturedProperties webProps, Str
       throws IOException, ClassCastException, ProtocolException {
 
     HttpURLConnection connection = (HttpURLConnection) webProps.url.openConnection();
+    connection.setReadTimeout(webProps.timeout);
```

Comment:
I think we want to set both `setConnectionTimeout` and `setReadTimeout`.

Reply:
Set both props